Prox/Prox/Data/TravelTimesProvider.swift

```diff
@@ -83,7 +83,7 @@ struct TravelTimesProvider {
     }
 
     static func canTravelFrom(fromLocation: CLLocationCoordinate2D, toLocation: CLLocationCoordinate2D, before: Date, withCompletion completion: @escaping (Bool) -> ()) {
-        let timeInterval = before.timeIntervalSince(Date())
+        let timeInterval = Date().timeIntervalSince(before)
```

Comment:
I'm going to strip out this change bc I'm not sure if we'll be rebasing off these changes.